payment/views.py

```diff
@@ -15,14 +17,31 @@ def payment_create(request):
             payment = form.save(commit=False)
             payment.ong = request.user.ong
             payment.save()
-            return redirect('payment_list')
+            return redirect('payment_create')
 
         else:
             messages.error(request, 'El formulario presenta errores')
     else:
         form = CreatePaymentForm()
 
-    return render(request, 'payment/payment_form.html', {'form': form, 'title': 'Añadir Pago'})
+        all_events = Payment.objects.all()
+        event_arr = []
+        for i in all_events:
+            event_sub_arr = {}
+            event_sub_arr['title'] = str(i.amount)
+            start_date = datetime.strptime(str(i.payday.date()), "%Y-%m-%d").strftime("%Y-%m-%d")
+            end_date = datetime.strptime(str(i.payday.date()), "%Y-%m-%d").strftime("%Y-%m-%d")
+            event_sub_arr['start'] = start_date
+            event_sub_arr['end'] = end_date
+            event_arr.append(event_sub_arr)
+        data = JsonResponse((event_arr), safe=False)
+        datatest = json.dumps(event_arr)
+            #return HttpResponse(json.dumps(event_arr))
+        print(data, type(data))
+        print(datatest, type(datatest))
+        #return HttpResponse(json.dumps(event_arr))
+
+    return render(request, 'payment/payment_form.html', {'form': form, 'title': 'Añadir Pago', 'events_json':datatest})
```

Comment:
You can pass the `context` variable with all those things to make more the code easier to read.